package.json

```diff
@@ -142,6 +142,7 @@
     "react-avatar-editor": "^12.0.0-beta.0",
     "react-dom": "^17.0.1",
     "react-grid-layout": "^1.1.1",
+    "react-input-date": "^1.0.2",
```

Comment:
who is managing the source code of this library? why don't we put it in github.com/openforis/...?